[2/n][a2][scripts] update scripts & add just cmds to run

xanderec opened the pull request:
Summary:

just commands to make life easier

moved basedir variable to .env file since it is different for each local machine

Test Plan:

just build
just run 1

---
[//]: # (BEGIN SAPLING FOOTER)
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/raydatray/comp-512/pull/87).
* #88
* __->__ #87

raydatray reviewed (approved):
shake it crazy style

Commit:
[2/n][a2][scripts] update scripts & add just cmds to run

Summary:

just commands to make life easier

moved basedir variable to .env file since it is different for each local machine

Test Plan:

just build
just run 1
